src/com/cft/contactmerge/MapMaker.java

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
I'm thinking ContactPropertyMatcher may be a better name.

Reply:
This class isn't actually matching properties to columns. It is creating the rules that ContactImporter (or a subclass/contained class) will use to do the matching.